filesystems/tests/common.py

```diff
@@ -73,14 +73,56 @@ def test_open_read_non_existing_nested_file(self):
             )
         )
 
+    def test_open_append_non_existing_file_as_bytes(self):
+        fs = self.FS()
+        tempdir = fs.temporary_directory()
+        self.addCleanup(fs.remove, tempdir)
+
+        with fs.open(tempdir.descendant("unittesting"), "ab") as f:
+            f.write(b"some ")
```

Comment:
It might be nice to use some actual non-ASCII text here to make sure we're not ever running into implicit encoding conversion on Py2

Reply:
#18